app/views/papyrus/_printers.html.slim

```diff
@@ -3,12 +3,11 @@
   = f.fields_for :preferred_printers, Papyrus::PreferredPrinter.new do |pf|
     .grid.grid-cols-12.gap-4
       .col-span-2
-        = pf.input :use, collection: Papyrus::Printer::USES, include_blank: false, as: :dropdown, text_method: :first, value_method: :last
+        = pf.input :use, collection: Papyrus::Printer::USES, include_blank: false, as: :dropdown
```

Comment:
this has been changed because this is no longer needed to add the text method and value method